src/python/turicreate/toolkits/sound_classifier/_tf_sound_classifier.py

```diff
@@ -0,0 +1,233 @@
+        'sound_dense1_bias'  : _tf.Variable(_tf.zeros([100]), name='sound_dense1_bias'),
+        'sound_dense2_bias'  : _tf.Variable(_tf.zeros([self.num_classes]), name='sound_dense2_bias')
+        }
+
+        dense0 = _tf.nn.xw_plus_b(self.x, weights=weights["sound_dense0_weight"], biases=biases["sound_dense0_bias"])
+        dense0 = _tf.nn.relu(dense0)
+
+        dense1 = _tf.nn.xw_plus_b(dense0, weights=weights["sound_dense1_weight"], biases=biases["sound_dense1_bias"])
+        dense1 = _tf.nn.relu(dense1)
+
+        out = _tf.nn.xw_plus_b(dense1, weights=weights["sound_dense2_weight"], biases=biases["sound_dense2_bias"])
+        out = _tf.nn.softmax(out)
+
+        self.predictions = out
+
+        # Loss
+        self.cost = _tf.reduce_mean(_tf.nn.softmax_cross_entropy_with_logits_v2(logits=self.predictions,
+            labels=self.y))
+
+        # Optimizer
+        self.optimizer = _tf.train.AdamOptimizer(learning_rate=0.01).minimize(self.cost)
+
+        # Predictions
+        correct_prediction = _tf.equal(_tf.argmax(self.predictions, 1), _tf.argmax(self.y, 1))
+        self.accuracy = _tf.reduce_mean(_tf.cast(correct_prediction, "float"))
+        
+        self.sess = _tf.Session()
+        self.sess.run(_tf.global_variables_initializer())
+
+        '''
+        # Assign the initialised weights from MXNet to tensorflow
+        layers = ['sound_dense0_weight', 'sound_dense0_bias', 'sound_dense1_weight', 'sound_dense1_bias',
+        'sound_dense2_weight', 'sound_dense2_bias']
+
+        for key in layers:
+            if 'bias' in key:
+                self.sess.run(_tf.assign(_tf.get_default_graph().get_tensor_by_name(key+":0"),
+                    net_params[key].data().asnumpy()))
+            else:
+                #if 'dense' in key:
+                self.sess.run(_tf.assign(_tf.get_default_graph().get_tensor_by_name(key+":0"),
+                                        net_params[key].data().asnumpy().transpose(1, 0)))
+        '''
+
+
+    def train(self, feed_dict):
+        _, final_train_loss, final_train_accuracy = self.sess.run([self.optimizer, self.cost, self.accuracy],
+                            feed_dict={
+                                self.x: feed_dict['x'],
+                                self.y: feed_dict['y']
+                            })
+        result = {'accuracy' : final_train_accuracy, 'loss' : final_train_loss}
+        return result
+
+    def predict(self, feed_dict):
+        pred_probs, final_accuracy = self.sess.run([self.predictions, self.accuracy],
+                            feed_dict={
+                                self.x: feed_dict['x'],
+                                self.y: feed_dict['y']
+                            })
+        result = {'accuracy' : final_accuracy, 'predictions' : pred_probs}
+        return result
+
+
+    def export_weights(self):
+        """
+        Retrieve weights from the TF model, convert to the format MXNET
```

Comment:
In general, export_weights is a function that converts the weights from existing format to CoreML format.

Reply:
For now, since we are not using this from C++, I think we want to match the serialization format currently used by the sound classification toolkit